flink-dist/src/main/resources/flink-conf.yaml

```diff
@@ -30,7 +30,7 @@
 # automatically configure the host name based on the hostname of the node where the
 # JobManager runs.
 
-jobmanager.rpc.address: localhost
+rest.address: localhost
```

Comment:
Wouldn't it make sense to also change `jobmanager.rpc.port` to `rest.port`?

Reply:
Good point!
I just find the deprecated chain `jobmanager.rpc.port→web.port→rest.port`, but we already have a config of `rest.port` at `flink-conf.yaml`. Suspect that `jobmanager.rpc.port` would not be respected at all.

Reply:
I am sorry that I misread the deprecated annotation. `rest.port` is the successor of `web.port` which is the successor of `jobmanager.web.port`. Not `jobmanager.rpc.port`.

Reply:
I guess it is because the previous jobmanager has been separated into RestServer/Dispatcher/JobMaster so that we now at a period where configuration is a bit mess?